test/e2e/performanceprofile/functests/1_performance/irqbalance.go

```diff
@@ -146,14 +145,8 @@ var _ = Describe("[performance] Checking IRQBalance settings", Ordered, func() {
 
 			defer func() { // return initial configuration
 				By("reverting the profile into its initial state")
-				spec, err := json.Marshal(initialProfile.Spec)
-				Expect(err).ToNot(HaveOccurred())
-				Expect(testclient.Client.Patch(context.TODO(), profile,
-					client.RawPatch(
-						types.JSONPatchType,
-						[]byte(fmt.Sprintf(`[{ "op": "replace", "path": "/spec", "value": %s }]`, spec)),
-					),
-				)).ToNot(HaveOccurred())
+				profile.Spec = initialProfile.Spec
+				Expect(testclient.ControlPlaneClient.Update(context.TODO(), profile)).ToNot(HaveOccurred())
```

Comment:
at glance the logic here seems questionable. We should `Get` in the loop, not just update. Let's address in a followup PR.